cli/src/commands/logs.ts

```diff
@@ -62,27 +65,50 @@ $ ${Command.cliBinName} logs --since 2m --follow`,
     }),
   };
 
-  public static args = [];
+  public static args = [
+    {
+      name: 'name',
+      required: false,
+      description: 'Application name (defaults to working directory\'s deployed application name)',
```

Comment:
(Not this PR)
It would be cool if I could run some command to determine this.  WDYT?

Reply:
makes sense. maybe even `reshuffle describe` which would describe the current app or the app by name